src/shared/utils.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Aliases } from '@salesforce/core';
+
+export const getAliasByUsername = async (username: string): Promise<string> => {
+  const alias = await Aliases.create(Aliases.getDefaultOptions());
+  const aliasContent = alias.getContents().orgs;
```

Comment:
Isn't this method the same as `https://forcedotcom.github.io/sfdx-core/classes/aliases.html#getkeysbyvalue`? I guess in this case you only want one, but an interesting question is what should happen if there is more than one alias? Currently, it only displays the first. Maybe it should display all of them though.

Reply:
I don't see that method, either in core OR in your link.

Reply:
As far as the multi-alias issue...this utils method could bring back multiple, but the list/display commands that use this don't currently support that and switch the property from string to string[] would probably be hard on the users relying on json output.

Maybe we should prevent multiple aliases from being created for the same username.

Reply:
I don't know. I can see multiple aliases on the same user being useful and since we already support that I wouldn't want to change it at this point.

This is how it currently works, so leave it. Just thought I would bring it up.

Reply:
What about the phantom `getkeysbyvalue` method?  Any idea where that is?

Reply:
@mshanemc it exists on Alias instance getKeysByValue
<img width="281" alt="Screen Shot 2021-01-20 at 1 26 54 PM" src="https://user-images.githubusercontent.com/17622511/105230834-34a2ce80-5b23-11eb-92b4-e3551c780cd5.png">

Reply:
Aliases extends ConfigGroup extends ConfigFile extends BaseConfigStore.  Inheritance...